gui/browserv7/src/RBrowser.cxx

```diff
@@ -464,13 +464,13 @@ std::string ROOT::Experimental::RBrowser::ProcessDblClick(const std::string &ite
       }
 
       // FIXME: how to proced with ownership here
-      TObject* clone = object->Clone();
-      TH1 *h1 = dynamic_cast<TH1*>(clone);
+      TObject *clone = object->Clone();
+      TH1 *h1 = dynamic_cast<TH1 *>(clone);
       if (h1) h1->SetDirectory(nullptr);
 
       std::shared_ptr<TObject> ptr;
       ptr.reset(clone);
-      rcanv->Draw<RObjectDrawable>(ptr, "");
+      rcanv->Draw<RObjectDrawable>(ptr, drawingOptions.c_str());
       rcanv->Modified();
```

Comment:
Here no need to use c_str() - just provide `drawingOptions`

Reply:
So i should remove it here too right ?
`canv->GetListOfPrimitives()->Add(object, drawingOptions.c_str());`

Reply:
ok, thanks